packages/create-app/templates/default-app/packages/app/src/components/Root/Root.tsx

```diff
@@ -71,9 +70,7 @@ export const Root = ({ children }: PropsWithChildren<{}>) => (
         <SidebarItem icon={CreateComponentIcon} to="create" text="Create..." />
         {/* End global nav */}
         <SidebarDivider />
-        <SidebarScrollWrapper>
-          <SidebarItem icon={MapIcon} to="tech-radar" text="Tech Radar" />
-        </SidebarScrollWrapper>
+        <SidebarScrollWrapper />
```

Comment:
since it's empty, maybe we can add a comment so that it is easier to understand what `SidebarScrollWrapper` is doing. It will also avoid collapsing the element.

```suggestion
        <SidebarScrollWrapper>
          { /* Items in this group will be scrollable if they run out of space */ }
        </SidebarScrollWrapper>
```

Reply:
I've made the requested update, although it's now failing the Windows `prettier:check` I think due to [this issue](https://github.com/prettier/prettier/issues/7825) , also discussed on SO [here](https://stackoverflow.com/questions/61958231/github-actions-prettier-finds-errors-only-on-windows-latest). I didn't want to be mucking about with the `.gitattributes` file with this PR, so your call on how to handle the failure.

Reply:
(Actually the prettier checks are failing on both Linux and Windows, while they check out fine when I run `prettier:check` locally....so not sure what's causing it to flag....)